src/build/genprod.scala

```diff
@@ -180,9 +180,9 @@ class Function(val i: Int) extends Group("Function") with Arity {
  *  object Main extends App {%s}
  *  }}}
  *
- *  Note that `Function1` does not define a total function, as might
+ *  Note that `Function$i` does not define a total function, as might
```

Comment:
I don't think this is right either.  The type signature of `PartialFunction` doesn't strongly suggest that it's applicable to `Function0` or `Function2`.  `Function0` doesn't even take any arguments!  What does it mean for it to not be total?  (That it simply doesn't work?)  I'd just leave it out in cases other than `Function1`.  Also, the wording isn't so great even for `Function1` in the original.